### ISSUE TITLE
When creating a new workspace, the Screen Reader (NVDA/JAWS) does not announce status message: A11y_ML.NET Model Builder_Create new workspace_ScreenReader

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3039]
### GitHubTags
#A11ySev2; #A11yTCS; #Model Builder - Machine Learning for .NET Developers; #DesktopApp; #Win32;#BM-MlnetModelBuilder-Win32-March25; #Win32;#WCAG4.1.3;#Status Messages;#A11YWCAG2.1;#A11yMAS; 

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment (Select Azure).
6. Tab till Set up workspace and press enter.
7. Tab till new workspace and press enter.
8. When creating a new workspace, the Screen Reader (NVDA/JAWS) does not announce status message.
 
### Actual Result:
When creating a new workspace, the Screen Reader (NVDA/JAWS) does not announce status message(creating workspace in azure, this make take few minutes.

### Expected Result:
The screen reader (JAWS/NVDA) should announce the status message "Creating a workspace in Azure" when it appears on the screen during the workspace creation process.

### User Impact:
Screen reader users won’t be able to know whether any action is performing, or workspace is creating or not.

### Attachment:

https://github.com/user-attachments/assets/00bbcf93-4e93-487b-b079-7278b7d504a6

**Comment by AnKushSingh05** [comment]
@zewditu 
Verified the issue in below environment.
Microsoft Visual Studio Enterprise 18 (64-bit) - main
Version 18.0.0 Preview 1.0 [10529.9.main]
Issue is still not fixed

![Image](https://github.com/user-attachments/assets/fb1b304d-020b-4ed7-952f-457a9d1a29ec)

**Comment by Mana-swi** [comment]
#a11ydb-status-not-conveyed;
